chainer/links/connection/convolution_2d.py

```diff
@@ -49,19 +53,15 @@ def __init__(self, in_channels, out_channels, ksize, stride=1, pad=0,
         W_shape = (out_channels, in_channels, kh, kw)
         super(Convolution2D, self).__init__(W=W_shape)
 
-        if initialW is not None:
-            self.W.data[...] = initialW
-        else:
-            std = wscale * numpy.sqrt(1. / (kh * kw * in_channels))
-            self.W.data[...] = numpy.random.normal(0, std, W_shape)
+        initializers.init_weight(self.W.data, initialW, scale=wscale)
```

Comment:
Thank you for pointing out. My idea is to change the argument of `scale` from `wscale` to `sqrt(wscale)`. In that case, the deviation of weights is proportional to the square root of `wscale`, even if users employ different initializers from Gaussian distribution. Do you think it is reasonable?